examples/resources/splight_dashboard_table_chart/resource.tf

```diff
@@ -0,0 +1,143 @@
+terraform {
+  required_providers {
+    splight = {
+      source  = "splightplatform/splight"
+      version = "~> 0.1.0"
+    }
+  }
+}
+
+resource "splight_dashboard" "DashboardTest_fz" {
+  name           = "DashboardTest"
+  related_assets = []
+}
+
+resource "splight_dashboard_tab" "DashboardTabTest_fz" {
+  name      = "TabTest"
+  order     = 0
+  dashboard = splight_dashboard.DashboardTest.id
+}
+
+resource "splight_dashboard_table_chart" "DashboardChartTest" {
+  name               = "ChartTest"
+  tab                = splight_dashboard_tab.DashboardTabTest.id
+  timestamp_gte      = "now - 7d"
+  timestamp_lte      = "now"
+  description        = "Chart description"
+  min_height         = 1
+  min_width          = 4
+  display_time_range = true
+  labels_display     = true
+  labels_aggregation = "last"
+  labels_placement   = "bottom"
+  show_beyond_data   = true
+  height             = 10
+  width              = 20
+  collection         = "default"
+
+  y_axis_unit        = "MW"
+  number_of_decimals = 4
+
+  chart_items {
+    ref_id           = "A"
+    type             = "QUERY"
+    color            = "red"
+    expression_plain = ""
+    query_filter_asset {
+      id   = "1234-1234-1234-1234"
+      name = "query filter asset name"
+    }
+    query_filter_attribute {
+      id   = "1234-1234-1234-1234"
+      name = "query filter attribute name"
+    }
+    query_plain = jsonencode([
+      {
+        "$match" = {
+          asset     = "1234-1234-1234-1234"
+          attribute = "1234-1234-1234-1234"
+        }
+      },
+      {
+        "$addFields" : {
+          "timestamp" : {
+            "$dateTrunc" : {
+              "date" : "$timestamp",
+              "unit" : "day",
+              "binSize" : 1
+            }
+          }
+        }
+      },
+      {
+        "$group" : {
+          "_id" : "$timestamp",
+          "value" : {
+            "$last" : "$value"
+          },
+          "timestamp" : {
+            "$last" : "$timestamp"
+          }
+        }
+      }
+    ])
+  }
+
+  chart_items {
+    ref_id           = "B"
+    color            = "blue"
+    type             = "QUERY"
+    expression_plain = ""
+    query_filter_asset {
+      id   = "1234-1234-1234-1234"
+      name = "query filter asset name"
+    }
+    query_filter_attribute {
+      id   = "1234-1234-1234-1234"
+      name = "query filter attribute name"
+    }
+    query_plain = jsonencode([
+      {
+        "$match" = {
+          asset     = "1234-1234-1234-1234"
```

Comment:
use the references

Reply:
Creating the asset and attributes too is like too much boilerplate for the example.. tho I can do it if you want.